templates/app/helpers/product_options_helper.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module ProductOptionsHelper
+  def product_variants_with_options
+    @product.variants_and_option_values_for(current_pricing_options)
+  end
+
+  def sorted_option_values(variant)
+    variant.option_values.sort_by { |value| value.option_type.position }.map(&:id)
+  end
+
+  def sorted_option_types(product)
+    product.option_types.sort_by(&:position)
+  end
+
+  # move to model
+  def option_values(option_type)
```

Comment:
I thought this method might make more sense to move into the product model, but never went back to move It. What are your thoughts on putting it into the product model?

Reply:
@cpfergus1 Hmmm.. Generally, it's good practice to limit access to a method only to the code that depend on it. As we broaden access to a method, we increase its cost of maintenance, since we have to check a wider area of code whenever we need to change the method.

In the case of the `option_values` method, if it can be reused in other areas of the codebase such as in the models or in the controllers, then yeah, it would make sense to move it to product model. 

However, at the moment, it seems like the method is only used in `product_selection/_option_type.html.erb`. Its logic is also very straightforward; we can get away without testing it :D Because of these factors, I think it might actually be better to inline the method in `product_selection/_option_type.html.erb`.

What do you think?